README.md

```diff
@@ -13,3 +14,21 @@
   complexity suggests a different time.
 
 Add your answers to this markdown file.
+
+**1. Asymptotic analysis disregards constants, which can make considerable differences. These constants are manifested in many different ways - different compilers, different machines, different programming languages, cache misses, etc.**
+
+**2. One reason that asymptotic analysis is misleading is manifested in the example of insertion sort. (O(n^2)) is the runtime complexity for the best, average, and worst case for insertion sort. In reality, however, there is a difference in the actual runtime of the algorithm when you have half the elements sorted (average case), versus when all of the elements are misplaced (worst case). In this regard, asymptotic analysis is misleading.**
```

Comment:
...but is this really because of differences in constants, e.g. the same as your first reason?

Reply:
My main point here is whether this is a new reason or the same as reason 1. You could either change this, or argue that your specific example of insertion sort amounts to more than just a constant.